contracts/src/v0.8/dev/ocr2dr/OCR2DROracle.sol

```diff
@@ -144,11 +144,11 @@ contract OCR2DROracle is OCR2DROracleInterface, OCR2Base, AuthorizedOriginReceiv
     bytes[] memory results;
     bytes[] memory errors;
     (requestIds, results, errors) = abi.decode(report, (bytes32[], bytes[], bytes[]));
-    if (requestIds.length != results.length && requestIds.length != errors.length) {
+    if (requestIds.length == 0 || (requestIds.length != results.length && requestIds.length != errors.length)) {
```

Comment:
Here we can OR all those three conditions - any of them failing indicates an invalid report.

Reply:
Good point. Changing.